### PR TITLE
wrote a task and added nil handling

### LINKED ISSUE #250: 500 on incidents show page if reason code is `nil`
https://github.com/umts/incidents/blob/25ead81ffb5aa884704d36ef991a7426b207ea03/app/views/incidents/show.html.haml#L32

A reason code isn't required for incidents; this line needs `nil`-handling.

### PR DESCRIPTION
**Pull request by avacmeyer**
closes #250 . I don't know how these incidents are getting completed but I guess this task can just get run again in the future.

### CODE REVIEW

File: lib/tasks/invalid_incidents.rake [hunk 1]
```diff
@@ -0,0 +1,12 @@
+namespace :incidents do
+  desc 'fix invalid incidents'
+  task update_incidents: :environment do
+    completed = Incident.where( completed: true )
+    completed.each do |incident|
+      if !incident.valid?
+        incident.completed = false
```
Comment (werebus) [inline lib/tasks/invalid_incidents.rake:7]: We should make sure to coordinate this with the affected staff. That is, they should be given a warning that they're about to have ~100 uncompleted incidents.

File: app/views/incidents/show.html.haml [hunk 2]
```diff
@@ -27,12 +27,14 @@
     %p
       %strong= human_name :incident, :root_cause_analysis
       = simple_format @incident.root_cause_analysis
-    %p
-      %strong= human_name :incident, :reason_code
-      = @incident.reason_code.full_label
-    %p
-      %strong= human_name :incident, :supplementary_reason_code
-      = @incident.supplementary_reason_code.full_label
+    - if @incident.reason_code_id?
```
Comment (werebus) [inline app/views/incidents/show.html.haml:30]: At this point, all completed, valid incidents will have `reason_code` and a `supplementary_reason_code`. You can probably lose these two conditionals.

**Review by werebus — CHANGES_REQUESTED** [review]

File: lib/tasks/invalid_incidents.rake (hunk 1 above)
Comment (avacmeyer) [reply lib/tasks/invalid_incidents.rake:7]: ooof yeah good point.

**Review by werebus — APPROVED** [review]
Seems fine to me.

Because it's a rake task, this can be merged safely at any point. I'll leave coordinating the *running* of said task to you, @avacmeyer

**Pull request merged**